config/initializers/doorkeeper.rb

```diff
@@ -58,6 +60,15 @@
   #     redirect_to sign_in_url
   #   end
   # end
+  #
+  # WARNING, the following code allow any user to edit Client Application endpoints!
+  admin_authenticator do
+    if current_user
```

Comment:
Can we simplify it to `current_user&.(is_admin?)`. Also we can add predicate `?` to method when using boolean values as per our conventions .(Rails add predicate method by default for boolean values).

Reply:
Good point for the `?` predicate @ankitkalia1195  :) 

Regarding `current_user&.is_admin?`, I think it's better to keep the 2 different paths:
1. If user is not logged = redirect to sign_in
2. If user is logged, then we check if he shall gets a 403 or not

As an unauthenticated admin user, it's more user-friendly :)

Reply:
Sure, makes sense :)

Reply:
Ok. predicate added. 
I keep this conversation unresolve so that @ankitkalia1195 you can confirm if my previous comment is ok for you, or not :) Thanks

Reply:
Meanwhile I saw your comment "sure makes sense", so I will mark the conversation as resolve :) Thanks! 🙏